### ISSUE TITLE
After installing the plugin, the settings are not accessible

### ISSUE THREAD

**Issue by paaskynen** [center-for-learning-management/moodle-local_courseexpiry#5]
On a Moodle 3.10.7 installation (PHP 7.3.29 ; MariaDB 10.5.12) after installing the plugin. There is no way to open that settings of the plugin. On the plugins page (Site administration - Plugins - Local Plugins) the entry for Course expiry is not a link to the settings page, but bold text. There is no error message. When I check the page source, the link to the settings page is there, but it does not display (in a Firefox browser).

**Comment by rschrenk** [comment]
Hello,
I am using it on a 3.10.6-Moodle. Don't think there should be a huge difference between those versions.
Can you please enable debugging in your config.php and report if you see any error messages?
Please add:

```
    @ini_set('display_errors', '1');
    $CFG->debugdisplay = true;
    $CFG->debug = 32767;
```



Kind regards

**Comment by paaskynen** [comment]
Thanks for your answer. This plugin meets a clear need, so I hope we can get it to work.

In our test site debugging was already at developer level and there are no error messages. The theme in use is Boost.

![localplugins](https://user-images.githubusercontent.com/15938712/134511771-2de43361-1670-40fb-b270-11023cfef44f.png)
Rgrds,
Paul.

**Comment by rschrenk** [comment]
Hello, we are also using boost_campus, so a child of boost. Which version of the plugin have you installed?

**Comment by paaskynen** [comment]
The version is 2021080700

**Comment by paaskynen** [comment]
And this is the page source for what my earlier pic displays. As you can see the text is outside the link code.

![source](https://user-images.githubusercontent.com/15938712/134513958-da98c0aa-ce0e-4032-a38c-f7beb1ec6db9.jpg)

**Comment by rschrenk** [comment]
This is weird - in my case, the link works.

Can you open the settings page directly? /admin/category.php?category=local_courseexpiry

**Comment by rschrenk** [comment]
please try to overwrite the settings.php with the most current one
https://github.com/center-for-learning-management/moodle-local_courseexpiry/blob/master/settings.php

**Comment by paaskynen** [comment]
Thanks for the tip. Now it looks like this. Is that what it is supposed to look like?

![localplugins2](https://user-images.githubusercontent.com/15938712/134518650-749e3136-e21d-4953-8c29-9066fc001ca3.jpg)

This implies that the version in the Moodle plugins database does not include this fix.

**Comment by rschrenk** [comment]
No, it should not be nested, but for some reason the nav tree is rendered differently in our Moodle versions. Until I find out what causes this, it is a workaround.
